pkg/cmd/build/build.go

```diff
@@ -130,8 +132,11 @@ func (ob *OktetoBuilder) buildWithOkteto(ctx context.Context, buildOptions *type
 	defer os.RemoveAll(secretTempFolder)
```

Comment:
I know it's not part of the change but perhaps it could be a quick win, if we make the folder name random too, we allow running multiple builds using the `--secret` flag. At the moment if I start two builds in parallel using `--secret`, the first that completes will wipe out the `.secret` folder

Reply:
I think this should be filed into an improvement issue. This can clash with how this feature works so lets file an issue. Can you add it?